internal/profiles/service.go

```diff
@@ -268,7 +268,7 @@ func (p *profileService) UpdateProfile(
 		minderv1.Entity_ENTITY_TASK_RUN:           profile.GetTaskRun(),
 		minderv1.Entity_ENTITY_BUILD:              profile.GetBuild(),
 	} {
-		if err = updateProfileRulesForEntity(ctx, ent, &updatedProfile, qtx, entRules, rules); err != nil {
```

Comment:
Note that this code already keeps `rule_instances` updated, so we still have all this information in that table. We are simply getting rid of the duplicated storage of that information in `entity_profile_rules`